homeassistant/components/deconz/device_trigger.py

```diff
@@ -319,20 +332,26 @@ def _get_deconz_event_from_device_id(hass, device_id):
     device_registry = await hass.helpers.device_registry.async_get_registry()
     device = device_registry.async_get(config[CONF_DEVICE_ID])
 
-    trigger = (config[CONF_TYPE], config[CONF_SUBTYPE])
+    triggers = (config[CONF_TYPE], config[CONF_SUBTYPE])
+
+    triggers = REMOTES[device.model][triggers]
 
-    trigger = REMOTES[device.model][trigger]
+    if not isinstance(triggers, dict):
```

Comment:
```suggestion
    if device.model not in (AQARA_CUBE_MODEL, AQARA_CUBE_MODEL_ALT1):
```

Reply:
This can be removed entirely if we change the triggers of all remotes {CONF_EVENT: x}